README.md

```diff
@@ -1,3 +1,8 @@
+<p align="center">
+  <img alt="NSIDC logo" src="https://nsidc.org/themes/custom/nsidc/logo.svg" width="150" />
+  <img alt="NSF logo" src="https://nsidc.org/sites/default/files/images/Logo/NSF.svg" width="150" />
+</p>
```

Comment:
Hmm, I hadn't noticed that. Hopefully not, if it's much smaller maybe we should adjust.

Reply:
Forced the size of each fig to be the same, but the NSF logo is still smaller because of padding around the edges of the image that the NSIDC logo lacks.

I don't think this is an issue either, so I'm going to mark as resolved.